impeller/renderer/backend/gles/sampler_gles.cc

```diff
@@ -74,7 +67,7 @@ bool SamplerGLES::ConfigureBoundTexture(const TextureGLES& texture,
   gl.TexParameteri(target.value(), GL_TEXTURE_MIN_FILTER,
                    ToParam(desc.min_filter, desc.mip_filter));
   gl.TexParameteri(target.value(), GL_TEXTURE_MAG_FILTER,
-                   ToParam(desc.mag_filter, MipFilter::kNone));
+                   ToParam(desc.mag_filter, MipFilter::kNearest));
```

Comment:
@chinmaygarde what is this supposed to be? This line seems sus before

Reply:
I'm not sure why I changed this when landing the blit pass... This might be debugging leftovers. I'd try changing it back and seeing if any of the GLES playgrounds don't draw anything once [this](https://github.com/flutter/engine/pull/40460#discussion_r1142695900) is addressed.

Reply:
Just reread this and realized `kNone` is actually correct here because of https://github.com/flutter/engine/pull/40460#discussion_r1142695900